### DISCUSSION TITLE
Notification popup at the wrong position

### DISCUSSION THREAD

**Discussion by victorsmits** [elementary/triage#297] [Q&A]
Hello I have setup a ElementaryOs 7 and I discover that the notification pop up where at the wrong position ( see screen bellow). Does someone knows how to deal with that ?

![image](https://github.com/elementary/triage/assets/22278315/e760c3ec-845c-4334-b440-b8e91c9771af)

**Comment by danirabbit** [comment]
Hey thanks for your report! This seems to be a duplicate of https://github.com/elementary/notifications/issues/218
